content/concepts/websockets.textile

```diff
@@ -568,11 +568,11 @@ Such concerns are deserving of an article all of their own, and you'll find plen
+As you have seen, there's a lot involved when implementing support for The WebSocket protocol, not just in terms of client and server implementation details, but also with respect to support for other transports to ensure robust support for different client environments, as well as broader concerns, such as "authentication and authorization":/documentation/core-features/authentication?utm_source=websockets&utm_medium=concepts, "guaranteed message delivery":https://support.ably.io/solution/articles/3000044640-message-durability-and-qos-quality-of-service-, "reliable message ordering":https://support.ably.io/solution/articles/3000044640-message-durability-and-qos-quality-of-service-, "historical message retention":/documentation/core-features/history?utm_source=websockets&utm_medium=concepts, and so forth.
 
-Ably is a battle-tested, enterprise-ready, cloud-based messaging platform with native support for WebSockets, "long polling":/concepts/long-polling?utm_source=websockets&utm_medium=concepts and other fallback transports, in addition to support for other "open protocols such as MQTT":https://www.ably.io/adapters?utm_source=websockets&utm_medium=concepts. We've already done all of the above work for you, which means you can get on with the real task of building your business's application platform and leave the mammoth task of implementing robust, reliable realtime messaging to us.
+Ably is a battle-tested, enterprise-ready, cloud-based messaging platform with native support for WebSockets, "long polling":/concepts/long-polling?utm_source=websockets&utm_medium=concepts and other fallback transports, in addition to support for other "open protocols such as MQTT":/adapters?utm_source=websockets&utm_medium=concepts. We've already done all of the above work for you, which means you can get on with the real task of building your business's application platform and leave the mammoth task of implementing robust, reliable realtime messaging to us.
 
-Find out more about the "Ably Realtime platform":https://www.ably.io/dsn?utm_source=websockets&utm_medium=concepts.
+Find out more about the "Ably Realtime platform":/dsn?utm_source=websockets&utm_medium=concepts.
```

Comment:
This actually needs to be the full URL because the page does not exist within /documentation

Reply:
gotcha, changing now...

Reply:
```suggestion
Find out more about the "Ably Realtime platform":https://www.ably.io/dsn?utm_source=websockets&utm_medium=concepts.
```

Reply:
@Nosfheratu you can accept my suggestion above and then we are good to go.